addon/components/imgix-image.js

```diff
@@ -1,9 +1,203 @@
-import Ember from 'ember';
-import ResizeMixin from 'ember-resize-mixin/main';
-import layout from '../templates/components/imgix-image';
-import ImgixPathBehavior from '../mixins/imgix-path-behavior';
-
-export default Ember.Component.extend(ResizeMixin, ImgixPathBehavior, {
-  layout: layout,
-  classNames: ['imgix-image-wrap']
+import Component from '@ember/component';
+import { computed, get, set } from '@ember/object';
+import ResizeAware from 'ember-resize-aware/mixins/resize-aware';
+import { merge } from '@ember/polyfills';
+import { tryInvoke } from '@ember/utils';
+import config from 'ember-get-config';
+import EmberError from '@ember/error';
+import ImgixClient from 'imgix-core-js';
+import { debounce } from '@ember/runloop';
+
+export default Component.extend(ResizeAware, {
+  tagName: 'img',
+  classNames: 'imgix-image',
+  attributeBindings: ['src', 'crossorigin', 'alt'],
+
+  path: null, // The path to your image
+  aspectRatio: null,
+  crop: 'faces',
+  fit: 'crop',
+  pixelStep: 1,
```

Comment:
This might be outside the scope of this PR, but I'm strongly opposed to setting the default `pixelStep` to 1: it's a really bad strategy from a caching perspective, and is going to result in unnecessarily long load times for many users. I'd recommend we set this to *at least* 10, and in the long-term I'd like to see us adopt a stepping approach more like the one we're using in imgix.js.

Reply:
Sounds good to me, I'll update the PR to `10`

Reply:
Done